### PR TITLE
Add payment contract functions - deploy and show

### PR DESCRIPTION
**Pull request by elv-serban**
As a tenant, create a payment contract for revenue splits.

At creation - specify the list of stakeholders:  an array of addresses and a corresponding array of shares.
Shares are integers and represent proportions so they don't need to, for example, add up to 100 (but they can, for simplicity).

### Create a payment contract

```
./elv-live payment_contract_create 0xc05e0274158442b7d595e5ac6d483d18df8fc93e,0x165b828643eb0fa8a7c6b15330670106169941cc 80,20
Deploy new payment contract

contract_address: '0xcd6c150e4da4a22cdc3d128f4024e039234d6b59'
shares: 100
```

### Show status of payment contract and stakeholders

```
./elv-live payment_contract_show 0xcd6c150e4da4a22cdc3d128f4024e039234d6b59 0x4c6C3547f81Ef23ED6e6562ab05a598DDdDd140d
Show payment contract status

contract_address: '0xcd6c150e4da4a22cdc3d128f4024e039234d6b59'
shares: 100
released: 0
payees:
  '0xC05e0274158442B7d595E5AC6D483d18DF8fc93e':
    shares: 80
    released:
      _hex: '0x00'
  '0x165B828643eB0fA8A7C6b15330670106169941cc':
    shares: 20
    released:
      _hex: '0x00'
```

## Follow up items (can be done as separate PRs)

- figure out how to call the payment 'releasable' function (currently fails)
- could refine the 'show' output to represent the values with 'decimals' instead of big ints

### CODE REVIEW

**Comment by elv-preethi** [comment]
`releasable` method was overloaded function. 
Followed fix from here:
https://github.com/ethers-io/ethers.js/issues/407
and made `formatArguments` false. It seems to work now.
```
./elv-live payment_contract_show 0xcd6c150e4da4a22cdc3d128f4024e039234d6b59 0x4c6C3547f81Ef23ED6e6562ab05a598DDdDd140d
Show payment contract status

contract_address: '0xcd6c150e4da4a22cdc3d128f4024e039234d6b59'
shares: 100
released: 0
payees:
  '0xC05e0274158442B7d595E5AC6D483d18DF8fc93e':
    shares: 80
    released: 0
    releasable: 0
  '0x165B828643eB0fA8A7C6b15330670106169941cc':
    shares: 20
    released: 0
    releasable: 0
```
@elv-serban

**Review by elv-serban — COMMENTED** [review]
Great find!

File: src/ElvContracts.js [hunk 1]
```diff
@@ -246,6 +246,132 @@ class ElvContracts {
     return {total: Ethers.BigNumber.from(res._hex).toNumber()};
   }
 
+  /**
+   * Deploy Payment contract (revenue splitter - commerce/Payment.sol)
+   * @param {string} addresses : list of stakeholder addresses
+   * @param {string} shares: list of stakeholder shares, in the order of addresses
+   */
+  async PaymentDeploy({ addresses, shares }){
```
Comment (elv-wayne) [inline src/ElvContracts.js:254]: Does the contract check for bad share values?

**Review by elv-wayne — APPROVED** [review]
Looks good, other than allowing shares to be non integer strings. Not sure how much input validation do we need on these contracts

File: src/ElvContracts.js (hunk 1 above)
Comment (elv-preethi) [reply src/ElvContracts.js:254]: These are the checks done in constructor:
```
require(payees.length == shares_.length, "PaymentSplitter: payees and shares length mismatch");
require(payees.length > 0, "PaymentSplitter: no payees");


require(account != address(0), "PaymentSplitter: account is the zero address");
require(shares_ > 0, "PaymentSplitter: shares are 0");
require(_shares[account] == 0, "PaymentSplitter: account already has shares");
```

Comment (elv-serban) [reply src/ElvContracts.js:255]: I added validation for addresses and shares (integers)

**Pull request merged**